packages/terra-button/src/Button.jsx

```diff
@@ -204,7 +204,12 @@ class Button extends React.Component {
     ]);
 
     const buttonText = !isIconOnly && variant !== 'utility' ? <span className={buttonTextClasses}>{text}</span> : null;
-    const buttonIcon = icon ? <span className={iconClasses}>{icon}</span> : null;
+
+    let buttonIcon = null;
+    if (icon) {
+      const cloneIcon = React.cloneElement(icon, { className: cx('icon-svg') });
```

Comment:
I would double check but I think this will overwrite any user provided classNames on the icon

Reply:
Yeah, we'll need to merge the classes here.